api/docs/intro.dox

```diff
@@ -587,6 +587,10 @@ heap.  The loader also attempts to isolate other global resource usage and
 global callbacks.  Please file reports on any transparency problems
 observed when using the private loader.
 
+If DynamoRIO and clients are statically linked into the application,
+the private loader is not available and third-party library usage is
+no longer safe (see \ref sec_static_DR).
```

Comment:
nit: maybe let's reiterate on the fact that they are still safe on initialization. Reading it like this sec_static_DR and this paragraph seem to contradict each other.
Something like: "the private loader is not available and third-party library usage is generally no longer safe, unless their usage happen at process-wide initialization (see \ref sec_static_DR)."

Reply:
Done.